python/run_look_with_filters.py

```diff
@@ -1,43 +1,53 @@
+import exceptions
 import json
 import sys
 from typing import cast, Dict, List, Union
 
-from looker_sdk import client, models
+from looker_sdk import client, models, error
 
-sdk = client.setup("../looker.ini")
+sdk = client.setup("looker.ini")
 
 
-def main():
+def main() -> None:
     """Given a look id, obtain the query behind it and run it with the desired
      filter values.
-    """
+
+    https://docs.looker.com/reference/api-and-integration/api-reference/v3.1/query#implementation_notes_9  
+    shows an example of how filters are defined in the posted body. To set the
+    same filter in this example, the script needs to be run as follows:
+
+    $ python run_look_with_filters.py lookId category.name socks
```

Comment:
since this is an example, the `lookId` should probably be a sample lookId value rather than the string, since the last 2 arguments are example values. Or:
```bash
$ python run_look_with_filters.py <lookId> <filterName> <filterValue>
```
e.g.  `python run_look_with_filters.py 5 category.name socks`